temporian/implementation/numpy/data/plotter.py

```diff
@@ -242,8 +245,12 @@ def plot(
         # Plot each feature individually
         >>> tp.plot(evset)
 
-        # Plots multiple features in the same sub-plot.
-        >>> tp.plot((evset,), interactive=True)
+        # Plots multiple features in the same sub-plot (equivalent).

```

Comment:
Remove `(equivalent)` ? I guess it's for the example below

Reply:
*Done